conditions/conditions_test.go

```diff
@@ -75,17 +76,17 @@ var _ = Describe("Condition", func() {
 	})
 
 	Describe("Get/Set", func() {
-		var operatorCond *apiv1.OperatorCondition
+		var operatorCond *apiv2.OperatorCondition
 
 		objKey := types.NamespacedName{
 			Name:      "operator-condition-test",
 			Namespace: ns,
 		}
 
 		BeforeEach(func() {
-			operatorCond = &apiv1.OperatorCondition{
+			operatorCond = &apiv2.OperatorCondition{
 				ObjectMeta: metav1.ObjectMeta{Name: "operator-condition-test", Namespace: ns},
-				Status: apiv1.OperatorConditionStatus{
+				Spec: apiv2.OperatorConditionSpec{
```

Comment:
a non-blocker, but it would be helpful to have [`conditionType`](https://github.com/operator-framework/api/blob/bb9b80e8278978efdb06a6f5d5682eb3cad330ec/pkg/operators/v1/operator_types.go#L19) declared in v2 too. Its odd that for someone using this library, they would have to use `conditionType` from `v1` and the OperatorCondition api from `v2`. Example - [here](https://github.com/operator-framework/operator-lib/blob/064486c30aa267578d6c552eb796ad99f969c213/conditions/conditions.go#L77).

If not the other option is - we can also create a follow up PR to remove the `conditionType` itself and let users specify a string directly.

Reply:
Addressed in this release: https://github.com/operator-framework/api/releases/tag/v0.9.1